api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -123,6 +123,17 @@ default void updateBillingStatus(long workspaceId, BillingStatus status) {
     save(toUpdate);
   }
 
+  default void updateInitialCreditsExpiration(long workspaceId, boolean expired) {
```

Comment:
can we remove this one too?

Reply:
Thanks. I replaced  this and the updateBillingStatus method.